src/dropfile/component.js

```diff
@@ -56,13 +56,12 @@ const fileDrop = function (gmfExternalDataSourcesManager, gettextCatalog) {
     /**
      * @param {angular.IScope} $scope Scope.
      * @param {JQuery} element Element.
-     * @param {angular.IAttributes} attrs Attributes.
      */
-    link: function ($scope, element, attrs) {
-      // @ts-ignore: scope ......
-      if ($scope.ctrl.fileDropEnabled !== true) {
-        return;
-      }
+    link: function ($scope, element) {
+      // // @ts-ignore: scope ......
+      // if ($scope.ctrl.fileDropEnabled !== true) {
+      //   return;
+      // }
```

Comment:
commented out because I get an "undefined" error for `$scope.ctrl`, but I wonder if we should drop this attribute (`fileDropEnabled`) altogether since the `gmf-file-drop-zone` component is now in the index.html and not in the map component?

Reply:
I think we can drop it. It looks to be there to (and I think I remember than it's there to) disable the drop on map where we don't want this feature. If it's in the index, we can already choose if we want this function or not.